.gitlab-ci.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-image: quay.io/govcms/govcms-ci
+image: govcms/govcms-ci:edge2
```

Comment:
https://github.com/govCMS/govcms-ci/pull/5

The above has been pushed as edge2 to circumvent some docker image caching in Gitlab CI. We will need to update this to latest when we merge https://github.com/govCMS/govcms-ci/pull/5.

Reply:
```suggestion
image: govcms/govcms-ci:latest
```